quantumdb-cli/src/test/resources/operations/CreateViewChangelog.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<changelog
+		xmlns="http://www.quantumdb.io/xml/ns/quantumdb-changelog"
+>
+
+	<changeset id="id1" author="Nick Richter">
+		<description>Test changeset 1</description>
+		<operations>
+			<createView viewName="view1">
+				<sql>
+					SELECT * FROM table1 WHERE admin = true;
```

Comment:
Does this not fail if the table doesn't exist?

Reply:
The unit tests will only test the Xml classes that make the XmlOperations. It will not check whether tables or columns exist.